### PR TITLE
Initial PR: connecting add page with BACKEND

### PR DESCRIPTION
**Pull request by hd-chuong**
Connect add page with BACKEND via `JQuery AJAX POST` methods.

**INVASIVE CHANGES**:
- Adding class name for each input in the table of `add.html` -> later fetch those inputs by vanillaJS `getElementsByClassName`

**New ADDING**:
- `add-new-record.php` 
      + receive POST data from add.js. 
      + Execute a backend validation to check if the data is invalid or missing.
      + Currently, product name has to be inputted with product number. Future features allow users to search for product name from a drop down list. 
      + Error messages can be viewed from browser `console`.

### CODE REVIEW

**Comment by nickmcm92** [comment]
PR moved to Draft as pending further work in ui-implementations branch

**Comment by hd-chuong** [comment]
This PR is ready to review now with previous commits of UI implementation is already merged to master.

File: src/backend_api/add-new-record.php [hunk 1]
```diff
@@ -17,16 +26,95 @@ function AddNewSalesRecord($table, $newRecord)
...
+            }
+            
+            // Lastly, safely remove the current sales record.
+            $salesRecordTable->delete($newRecordID);
+            die("Can not insert rows. Cancel adding. Remove previous added details.");
+        }
+    }
 
-    echo $last_id;
+    echo SUCCESS_CODE;
```
Comment (nickmcm92) [inline src/backend_api/add-new-record.php:119]: We should consider setting the HTTP Status code.

**Review by nickmcm92 — APPROVED** [review]
We should consider returning HTTP status codes.
Currently the form submission does not redirect. Once the view sales record is created, we should redirect to our newly created record.

**Pull request merged**